packages/conform-zod/v4/constraint.ts

```diff
@@ -106,6 +106,9 @@ export function getZodConstraint(schema: $ZodType): Record<string, Constraint> {
 		} else if (def.type === 'optional') {
 			constraint.required = false;
 			updateConstraint(def.innerType, data, name);
+		} else if (def.type === 'nullable') {
+			constraint.required = false;
```

Comment:
`nullable` allows null values, it doesn't make them optional, so this line shouldn't be necessary.